lib/flux-core/src/iter/adapters/skip.rs

```diff
@@ -0,0 +1,14 @@
+use flux_attrs::*;
```

Comment:
I don't think we are using this in any test; we should either add a test or remove it.

And talking about tests, I added support for writing tests directly in the modules where we add the extern specs (see `slice/mod.rs` for example). My rationale was that it would be better to have them close to the specs. But it is a bit more complicated to write tests like that so we could move them to the normal test infrastructure.